draftjs_exporter/wrapper_state.py

```diff
@@ -128,7 +128,16 @@ def element_for(self, block):
         options = Options.for_block(self.block_map, type_)
 
         # Make an element from the options specified in the block map.
-        elt = DOM.create_element(options.element[0], options.element[1])
+        tag = options.element[0]
+        attributes = options.element[1]
+
+        if (hasattr(tag, 'render')):
+            # If element wrapper is specified as object with 'render' method
+            attributes['data'] = block.get('data')
```

Comment:
I'm wondering what should happen if there is data on a block, but the block type isn't mapped to a component (eg. `DirectSpeech` in your example).

- Should that be considered an error, and raise an exception?
- Should the data be discarded silently?
- Should we attempt to add the data as attributes on the tag?

The reason I ask is that here you only do `attributes['data'] = block.get('data')` if the block has some sort of component – if this behaviour is desirable, I think we might need to introduce more logic either in `DOM.create_element` to handle this case.